nixos/modules/misc/wordlist.nix

```diff
@@ -0,0 +1,50 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+# Processes the list of paths
+# Returns either the sole path passed in if it's a singleton list.  Otherwise
+# creates a wordlist constructed from the various paths.
+processList = name: paths:
+  if builtins.length paths == 1
+  then
+    head paths
```

Comment:
this will probably not work if a user sets eg `environment.worlists.WORDLIST = [ ./wordlist ]` because paths themselves aren't strings. both legs if this condition will need a `toString`, so it might make sense to move the toString into the mapAttrs below instead.

Reply:
> did the enable option go away? it would be very good to have one for this module, scowl would add over 100MB to our system closure size for no tangible benefit to us.
> 
> indentation of stuff in the top-level let is also a bit wonky, usually let bindings are indented by two spaces. but all in all this is shaping up nicely :)

Sorry, I removed the `enable` option, because it seemed redundant to have "enable this" along with "use this list of environment variables".  I should have also emptied the default as well, but it slipped my mind.